static/script/devices/mediaplayer/live/restartable.js

```diff
@@ -56,7 +56,12 @@ require.def(
             },
 
             beginPlayback: function() {
-                this._mediaPlayer.beginPlayback();
+                var config = RuntimeContext.getDevice().getConfig();
+                if (config && config.forceBeginPlaybackToEndOfWindow) {
```

Comment:
Tried and failed to comment on this before...

Do we really want this to be a top-level configuration item? The other config option used by the media API is in streaming -> overrides.